schema/release-schema.json

```diff
@@ -122,6 +122,14 @@
       "description": "Information from the planning phase of the contracting process. Note that many other fields can be filled in a planning release, in the appropriate fields in other schema sections; these would likely be estimates at this stage, e.g. value in tender.",
       "type": "object",
       "properties": {
+        "id": {
+          "title": "Planning ID",
+          "description": "An identifier for this planning stage.",
```

Comment:
Is there a particular reason to use 'stage' rather than 'process' (as in the previous commit)?

According to https://github.com/open-contracting/standard/issues/866, in 1.2 the guidance will be that the `planning` section should only appear in planning processes, which are defined separately from contracting processes.

Reply:
> Is there a particular reason to use 'stage' rather than 'process' (as in the previous commit)?

For consistency. Planning description uses "planning phase", Planning milestones "planning stage", but planning documents "planning process".  But you are right, happy to change it back to process, and maybe update the other fields' descriptions to process as well?

Reply:
Aha, I see. Yes, let's go with "planning process" here, since the three terms are used equally and interchangeably and "planning process" aligns better with 1.2. For changing the other descriptions, maybe you can open a new issue about ensuring that the terminology in the planning section makes sense for planning processes (similar to https://github.com/open-contracting/standard/issues/1154) in case we need to make any other changes.

Reply:
I've updated all the references "planning" in the release schema now